cli/clibase/cmd.go

```diff
@@ -170,6 +172,7 @@ func (c *Cmd) Invoke(args ...string) *Invocation {
 		Stdout:  io.Discard,
 		Stderr:  io.Discard,
 		Stdin:   strings.NewReader(""),
+		Logger:  slog.Make(),
```

Comment:
At least for tests, this could be a named logger based on the command being run? Might help when there are multiple commands being run.

Reply:
Right now, it's just called "cli" for any Invocations created by `clitest.New()`.  You can override it in any complex test by setting `inv.Logger` before starting the command.